src/lib/components/Circos.react.js

```diff
@@ -15,10 +15,25 @@ export default class Circos extends Component {
     constructor(props) {
         super(props);
         this.circos = null;
-        this.configIdeogram = this.configIdeogram.bind(this);
```

Comment:
Oh, I see. Super weird that this was in here to begin with.

Reply:
lol i just assumed that that was meant to be `circos` not `ideogram`